test/integration/depth_camera.cc

```diff
@@ -678,6 +678,69 @@ void DepthCameraTest::DepthCameraParticles(
     EXPECT_LT(pointParticleAvg, pointAvg);
     EXPECT_LT(depthParticleAvg, depthAvg);
 
+    // test setting particle scatter ratio
+    // reduce particle scatter ratio - this creates a "less dense" particle
+    // emitter so we should have larger depth values on avg since fewers
+    // depth readings are occluded by particles
```

Comment:
If we reduce the scatter ratio, doesn't that mean there is less scatter, which means that the emitter is actually more dense, so depth values on average would be less? Based on my understanding/interpretation, the scatter ratio essentially determines how much scatter exists in the particles (lower ratio = less scatter, higher ratio = more scatter). I left another comment about documenting the behavior/interpretation of the scatter ratio parameter, hopefully my interpretation was correct there.

Reply:
> If we reduce the scatter ratio, doesn't that mean there is less scatter, which means that the emitter is actually more dense
the emitter would actually be less dense - more rays will be casting through the particles (due to less occlusion). So the average depth value will be higher (longer rays).  On the other hand, a higher scatter ratio means more dense and more rays will be bounced back by the particles, resulting in shorter depth values. I've tweaked the description to clarify how this param works.

Reply:
That makes sense to me. Thanks for the explanation!